romanesco/plugins/spark/pyspark_executor.py

```diff
@@ -6,7 +6,7 @@
 def run(task, inputs, outputs, task_inputs, task_outputs, **kwargs):
     from . import SC_KEY
 
-    custom = imp.new_module("custom")
+    custom = imp.new_module("__romanesco__")
```

Comment:
Makes sense, I agree. I'll give this the LGTM since nobody seems to have objected.